Tests

```diff
@@ -0,0 +1,94 @@
+Test Cases for Fakestagram:
+
+Note: For all necessary test cases, the User that can be used will have the following information:
+User 1:
+Email Address: test@test.com
+		Password: testtest
+User 2:
+Email Address: test2@test.com
+		Password: test2test2
+
+
+1.	Test login with correct user fields
+•	Acting as a User, 
+•	Visit the Login page
+•	Fill in all the input fields with the same details as initialised by the User
+•	Click on the login button
+•	Assert true if it redirects you to the home page
+
+2.	Test login with incorrect user fields
```

Comment:
Is it possible tests 1 & 2 could be seen as the same test by TA's?
(since though it is different test cases, it tests the behaviour of the same input group.)